### DISCUSSION TITLE
按照快速开始中docker部署的，配置httpbin服务无返回

### DISCUSSION THREAD

**Discussion by TanXiang7o** [higress-group/higress#1037] [Q&A] [answered]
很困惑，重试了几十遍了，劝退啊
![image](https://github.com/alibaba/higress/assets/82364837/a2e09336-456f-4807-a690-a60e1fa9e7a8)
![image](https://github.com/alibaba/higress/assets/82364837/2f6dba6b-d34b-4067-ab10-9446304c9dfb)
![image](https://github.com/alibaba/higress/assets/82364837/cd14447d-0c78-413d-95bc-fe1da80ca112)
![image](https://github.com/alibaba/higress/assets/82364837/e7b8df3f-8038-40f3-8524-a603e30f2089)
![image](https://github.com/alibaba/higress/assets/82364837/83ea1ef0-fa7f-40b7-8bf4-41c313df6800)
![image](https://github.com/alibaba/higress/assets/82364837/ca1947af-25a6-4ed4-a7d3-20039d468aa9)

**Comment by CH3CHO** [comment, marked as answer]
curl 加上 -v 参数看看。另外，根据你的截图，判断你用的是 WSL。Windows 下 80 端口是不是已经被别的人监听了呢？建议你改一下参数换个端口。
